_posts/2022-04-22-announcing-djangocon-us-2022.md

```diff
@@ -0,0 +1,26 @@
+---
+author: DjangoCon US Organizers
+category: General
+date: 2022-04-22 06:00:00
+layout: post
+image: /static/img/blog/announcing-2019.png
+title: "Ready, Set, Here We Go Again!"
+published: false
+---
+Hello, everyone! We are very excited to announce that the website for DjangoCon US 2022 is [live](https://2022.djangocon.us). Many thanks to [YupGup](http://yupgup.com/) for the website. It looks amazing!
+
+This year the conference will be October 16-21.  
+
+- October 16: Tutorials (Additional fee)
+- October 17-19: Talks
+- October 20-21: Sprints
+
+This year we will be returning to the [Marriott of Mission Valley](https://2022.djangocon.us/venue/) in San Diego, CA. Tickets are [on sale now](https://ti.to/defna/djangocon-us-2019) and our [hotel block is open](https://bit.ly/2UZ0oQQ). We sold out last year, so take advantage of our early bird pricing! If you’re looking for a ride or room share, [check out this spreadsheet](https://docs.google.com/spreadsheets/d/1-O7vSIeSls_xH9w9QVjVrVjgmEciKGduKQRMocCskkk/edit?usp=sharing).
+
+We will open our [opportunity grant](https://2019.djangocon.us/opportunity-grants/) (formerly financial aid) application on April 1.
+
+Now is the time to start getting your submissions for talks and tutorials ready because the [call for proposals (CFP)](https://2022.djangocon.us/speaking/) will open on April 1. We look forward to seeing all of the prospective talks and tutorials!
```

Comment:
```suggestion
Now is the time to start getting your submissions for talks and tutorials ready because the [call for proposals (CFP)](https://2022.djangocon.us/speaking/) will open on April 25. We look forward to seeing all of the prospective talks and tutorials!
```